pkg/kubelet/cm/memorymanager/memory_manager_test.go

```diff
@@ -2144,14 +2136,32 @@ func TestGetTopologyHints(t *testing.T) {
 				containerRuntime: mockRuntimeService{
 					err: nil,
 				},
-				activePods:        func() []*v1.Pod { return nil },
+				activePods: func() []*v1.Pod {
+					return []*v1.Pod{
+						{
+							ObjectMeta: metav1.ObjectMeta{
+								UID: "fakePod1",
+							},
+							Spec: v1.PodSpec{
+								Containers: []v1.Container{
+									{
+										Name: "fakeContainer1",
+									},
+									{
+										Name: "fakeContainer2",
+									},
+								},
+							},
+						},
+					}
+				},
```

Comment:
I think we should leave the old logic in place (to continue testing with no active pods list), and then add this as a second `Run()` loop in this function to test with something in the active pods list (and an admitting pod).